docs/platforms/javascript/common/migration/v7-to-v8/v8-new-performance-api.mdx

```diff
@@ -0,0 +1,253 @@
+---
+title: New Performance Monitoring APIs
+sidebar_order: 8997
+description: "Learn about new Performance Monitoring APIs in Sentry JavaScript SDK 8.x"
+---
+
+The SDK `8.x` release introduces new APIs for performance monitoring. These APIs are designed to provide more control over how performance data is collected and reported to Sentry.
+
+These APIs are also available in `7.x` so you can incrementally moveover to the new APIs before upgrading to `8.x`.
+
+Previously, there were two key APIs for adding manual performance instrumentation to your applications:
+
+- `startTransaction()`
+- `span.startChild()`
+
+This showed the underlying data model that Sentry was originally based on, which is that there is a root Transaction which can have a nested tree of Spans.
+
+In the new model, transactions are conceptually gone. Instead, you will always operate on spans, no matter where in the tree you are. Note that in the background, spans may still be grouped into a transaction for the Sentry UI. However, this happens transparently, and from an SDK perspective, all you have to think about are spans.
```

Comment:
In the new model, transactions are conceptually gone. Now, operations are always performed on spans, regardless of their position in the tree. Note that spans may still be grouped into a transaction for the Sentry UI. However, this happens in the background and, from the SDK's point of view, all you need to be concerned about are spans.

Reply:
```suggestion
In the new model, transactions are conceptually gone. Now, operations are always performed on spans, regardless of their position in the tree. Note that spans may still be grouped into a transaction for the Sentry UI. However, this happens in the background and, from the SDK's point of view, all you need to be concerned about are spans.
```